centreon/www/include/monitoring/downtime/downtime.php

```diff
@@ -82,7 +82,15 @@
         } else {
             unvalidFormMessage();
         }
-        require_once($path . "listDowntime.php");
+        try {
+            require_once($path . "listDowntime.php");
+        } catch (Exception $ex) {
+            CentreonLog::create()->error(
+                logTypeId: CentreonLog::TYPE_BUSINESS_LOG,
+                message: $ex->getMessage(),
```

Comment:
Can you write a message with the dowtime context please to have more details of business context ? I suggest for example : 
```suggestion
                message: "Error while listing downtime : $ex->getMessage()",
```

Reply:
thanks for your changes